builtInFunctions/esdtDataStorage_test.go

```diff
@@ -615,19 +615,12 @@ func TestEsdtDataStorage_SaveESDTNFTToken(t *testing.T) {
 		nonce := uint64(10)
 		key := baseESDTKeyPrefix + tokenIdentifier
 		tokenKey := append([]byte(key), big.NewInt(int64(nonce)).Bytes()...)
-		setTokenTypeCalled := false
 
 		args := createMockArgsForNewESDTDataStorage()
 		args.GlobalSettingsHandler = &mock.GlobalSettingsHandlerStub{
 			GetTokenTypeCalled: func(esdtTokenKey []byte) (uint32, error) {
 				return uint32(core.NonFungibleV2), nil
 			},
-			SetTokenTypeCalled: func(esdtTokenKey []byte, tokenType uint32) error {
```

Comment:
why did you delete the test ?

Reply:
This call was redundant. In the normal flow, `SetTokenType` is called by `esdtSetTokenType` and the token type is set inside `globalSettingsHandler`. Here we should set the type from `globalSettingsHandler` inside `esdtData` in order to be consistent, so no `SetTokenTypeCalled` is needed